### DISCUSSION TITLE
Welcome to nestup_evn Discussions!

### DISCUSSION THREAD

**Discussion by trvqhuy** [trvqhuy/ha-evn#4] [Announcements]
## 👋 Xin chào mọi người!
Từ giờ trở đi, chúng ta sẽ sử dụng phần Discussion này để kết nối cộng đồng HA Việt Nam sử dụng integration **nestup_evn**. 
Hy vọng là các bạn sẽ:
  * Thảo luận những câu hỏi mà bạn thắc mắc về integration. 
  * Chia sẻ những ý tưởng cải tiến/cải thiện.
  * Kết nối với tất cả mọi người thuộc cộng đồng HA Việt Nam.
  * Mọi người cứ tự nhiên đăng bài tại Discussion này. 

Hãy nhớ đây là một dự án vì cộng đồng 💪.

**Comment by leducanh0810** [comment]
Chào Huy, cám ơn Huy đã hỗ trợ cộng đồng ở phần code đọc dữ liệu của EVN.
Mình đang sử dụng và nhờ Huy có thể hỗ trợ đọc thêm các giá trị sau ở EVNCPC được không:
- Chỉ số ĐMTMN đầu kỳ (kWh)
- Chỉ số ĐMTMN đọc gần nhất (kWh)
- Sản lượng ĐMTMN đọc gần nhất (kWh)
Cám ơn bạn

**Comment by haind13** [comment]
Góp idea:
1. Cách tính giá điện cũng hay nhưng chưa thực tế và sát lắm, góp ý là xem xét để áp công thức bậc thang cho phù hợp 
--> mà để tính đc thì cần phải có dữ liệu đủ từ ngày mở hóa đơn mới 
2. Thể hiện số điện tiêu thụ theo biểu đồ cột hàng ngày sẽ dễ theo dõi hơn

**Comment by xiaomihomeDomo** [comment]
Hello, I use your module, everything works fine, I retrieved the information on HA with my EVN codes. 
I would like to have a daily consumption graph, like on your screenshort.
How can I do ? 
Thank you.

https://github.com/trvqhuy/nestup_evn/blob/main/screenshots/ui_display.png

**Comment by leolionart** [comment]
Có cách nào để chạy service trực tiếp trên linux để đẩy qua Grafana không nhỉ. Mình không xài HA nhưng muốn xài cái này
